myhpi/static/js/navbar.js

```diff
@@ -17,7 +17,7 @@
  * @returns True if it is collapsed, false otherwise.
```

Comment:
If you used some specifc linter (settings), can you include them in the README?

Reply:
For JS I can, but for CSS/HTML I'd rather get those in return ^^

Reply:
Added prettier into the pre-commit hook